Cargo.toml

```diff
@@ -36,5 +36,5 @@ serde_repr = "0.1.18"
 wasm-bindgen-test = "0.3.42"
 
 [features]
-# Enables the wgpu feature on vello, which is disabled by default
+default = []
```

Comment:
Hmm, how can this be 0.1.1 if by default we no longer enable the wgpu feature?

Reply:
I put this comment on vello_svg also, but

> I don't want to do semver with this- I want to keep parity with Vello.
> In other words, vello_svg 0.1.X pairs with Vello 0.1
> Semver also really only applies after 1.0